src/index.js

```diff
@@ -1359,15 +1359,18 @@ export class ULabel {
     }
 
     handle_toolbox_overflow() {
-        let tabs_height = $("#" + this.config["container_id"] + " div.toolbox-tabs").height();
-        $("#" + this.config["container_id"] + " div.toolbox_inner_cls").css("height", `calc(100% - ${tabs_height + 38}px)`);
-        let view_height = $("#" + this.config["container_id"] + " div.toolbox_cls")[0].scrollHeight - 38 - tabs_height;
-        let want_height = $("#" + this.config["container_id"] + " div.toolbox_inner_cls")[0].scrollHeight;
-        if (want_height <= view_height) {
-            $("#" + this.config["container_id"] + " div.toolbox_inner_cls").css("overflow-y", "hidden");
-        }
-        else {
-            $("#" + this.config["container_id"] + " div.toolbox_inner_cls").css("overflow-y", "scroll");
+        try {
+            let tabs_height = $("#" + this.config["container_id"] + " div.toolbox-tabs").height();
+            $("#" + this.config["container_id"] + " div.toolbox_inner_cls").css("height", `calc(100% - ${tabs_height + 38}px)`);
+            let view_height = $("#" + this.config["container_id"] + " div.toolbox_cls")[0].scrollHeight - 38 - tabs_height;
+            let want_height = $("#" + this.config["container_id"] + " div.toolbox_inner_cls")[0].scrollHeight;
+            if (want_height <= view_height) {
+                $("#" + this.config["container_id"] + " div.toolbox_inner_cls").css("overflow-y", "hidden");
+            } else {
+                $("#" + this.config["container_id"] + " div.toolbox_inner_cls").css("overflow-y", "scroll");
+            }
+        } catch (e) {
+            console.log("[WARN]: Failed to resize toolbox", e);
```

Comment:
```suggestion
            console.warn("Failed to resize toolbox", e);
```
https://developer.mozilla.org/en-US/docs/Web/API/console/warn_static